scripts/valhalla_build_config

```diff
@@ -118,11 +118,45 @@ config = {
         'max_reserved_labels_count_bidir_astar': 1000000,
         'max_reserved_labels_count_dijkstras': 4000000,
         'max_reserved_labels_count_bidir_dijkstras': 2000000,
-        'costmatrix_check_reverse_connection': False,
-        'costmatrix_allow_second_pass': False,
-        'max_reserved_locations_costmatrix': 25,
         'clear_reserved_memory': False,
         'extended_search': False,
+        'costmatrix': {
+            'check_reverse_connections': False,
```

Comment:
we should have done this from the beginning... the bad news is that those have defaults in code, we should probably remove those so users are forced to update.